playground/src/components/Header.tsx

```diff
@@ -4,22 +4,22 @@ import { Dialog } from '@headlessui/react';
 import {
   Bars3Icon,
   XMarkIcon,
-  QuestionMarkCircleIcon
+  ArrowUpRightIcon
 } from '@heroicons/react/24/outline';
 
 /** External links for the top navigation menu */
 const navigation = [
   {
-    name: 'Documentation',
-    href: 'https://0xpolygonmiden.github.io/miden-vm/intro/main.html'
+    name: 'HOME',
+    href: 'https://polygon.technology/solutions/polygon-miden/'
   },
   {
-    name: 'Developer Tools',
-    href: 'https://0xpolygonmiden.github.io/miden-vm/tools/main.html'
+    name: 'DOCUMENTATION',
```

Comment:
Can you change the documentation link to 

https://docs.polygon.technology/miden/